### PR TITLE
Issue 15 new delay strategy

### PR DESCRIPTION
**Pull request by bartdag**
Added alternative to celery's eta implementation. The doc explains the feature.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -205,6 +205,95 @@ This will override small parts of the django-celery Admin to enable the
...
+
+Semantics
+~~~~~~~~~
+
+The task is guaranteed to:
+
+1. Be sent at most once.
+2. Be sent after the ETA has expired (i.e., not before)
+
+If a crashes occur before a task is fully sent, the state of the scheduled task
```
Comment (ginmrt) [inline README.rst]: If a crash

File: django_celery_fulldbresult/__init__.py [hunk 2]
```diff
@@ -1,35 +1,74 @@
...
 
 from django.conf import settings
 from django.utils.timezone import now
 
 
-if getattr(settings, "DJANGO_CELERY_FULLDBRESULT_TRACK_PUBLISH", False):
+old_apply_async = Task.apply_async
+
+
+def new_apply_async(
```
Comment (ginmrt) [inline django_celery_fulldbresult/__init__.py]: single line?

File: django_celery_fulldbresult/__init__.py [hunk 3]
```diff
@@ -1,35 +1,74 @@
...
+        self, *args, **kwargs):
+    try:
+        return old_apply_async(self, *args, **kwargs)
+    except SchedulingStopPublishing as exc:
+        #
```
Comment (ginmrt) [inline django_celery_fulldbresult/__init__.py]: Forgot a comment?

File: django_celery_fulldbresult/tasks.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+from django_celery_fulldbresult.models import (
+    TaskResultMeta, SCHEDULED, SCHEDULED_SENT)
+
+
+@shared_task(ignore_result=True)
+def send_scheduled_task():
+    """Task that sends due scheduled tasks for execution.
+
+    Each DB operation must be in its own commit so that even if
+    sent_scheduled_task is called multiple times, a task is ensured to only be
```
Comment (ginmrt) [inline django_celery_fulldbresult/tasks.py]: send_scheduled_task

File: README.rst [hunk 5]
```diff
@@ -205,6 +205,95 @@ This will override small parts of the django-celery Admin to enable the
...
+Configuration
+~~~~~~~~~~~~~
+
+Set this variable in your settings.py file:
+
+::
+
+    DJANGO_CELERY_FULLDBRESULT_SCHEDULE_ETA = True
+
+Then create a periodic task in the Django admin or within your code. For
```
Comment (ginmrt) [inline README.rst:245]: Could that be included as part of a migration?

**Comment by ginmrt** [comment]
looking good

File: README.rst (hunk 5 above)
Comment (bartdag) [inline README.rst:245]: Maybe I could create a command that creates a periodic task instead. The thing is that I don't want to create a periodic task in somebody else system without their consent. Also, if the setting is False, then no task should be created.

**Pull request closed without merging**